src/app/features/collectibles/ordinals.tsx

```diff
@@ -11,44 +10,6 @@ interface OrdinalsProps {
 export function Ordinals({ query }: OrdinalsProps) {
   const { isLoading, isError, data } = query;
```

Comment:
Could we pass in only the props we need/be explicit, rather than the whole query response? 

Data is a bad name, and we're passing in many unused props.

Reply:
We can, the tradeoff is that we'll loose type inference: TS no longer understands that if `isError` is falsy, `data` is defined. We'll have to add an extra undefined check to `data` or get creative with types. I don't really see the harm in using it this way, but if you feel strongly about it I'll change it.

Sure, can rename `data` to `ordinals` for example

Reply:
Update: this is going to get refactored soon anyways.